.github/workflows/scripts/inspect_remote_img.sh

```diff
@@ -1,24 +1,23 @@
 #!/bin/bash
 
 inspect_remote_img() {
-  if [[ $# -ne 2 ]]; then
-    echo 'inspect_remote_img $GH_TOKEN $IMAGE'
-    echo 'Example: inspect_remote_img XXXXXXXXXXXX ghcr.io/nvidia/t5x:latest'
+  if [[ $# -ne 4 ]]; then
+    echo 'inspect_remote_img $GH_TOKEN $IMAGE $OS $ARCH'
+    echo 'Example: inspect_remote_img XXXXXXXXXXXX ghcr.io/nvidia/upstream-t5x:latest linux amd64'
     echo 'Returns the metadata of a tagged remote image (no download)'
     return 1
   fi
   GH_TOKEN=$1
   IMAGE=$2
+  OS=$3
+  ARCH=$4
 
-  TAG=$(echo $IMAGE | rev | cut -d: -f1 | rev)
   IMAGE_REPO=$(echo $IMAGE | rev | cut -d: -f2- | rev)
 
   PACKAGE=$(echo $IMAGE_REPO | rev | cut -d/ -f1 | rev)
   ORG=$(echo $IMAGE_REPO | rev | cut -d/ -f2 | rev)
 
-  top_manifest_digest=$(curl -s -H "Authorization: Bearer $(echo $GH_TOKEN | base64)" "https://ghcr.io/v2/$ORG/$PACKAGE/manifests/$TAG" | jq -r .manifests[0].digest)
-  config_digest=$(curl -s -H "Authorization: Bearer $(echo $GH_TOKEN | base64)" "https://ghcr.io/v2/$ORG/$PACKAGE/manifests/$top_manifest_digest" | jq -r .config.digest)
-  img_metadata=$(curl -s -L -H "Authorization: Bearer $(echo $GH_TOKEN | base64)" "https://ghcr.io/v2/$ORG/$PACKAGE/blobs/$config_digest")
-  echo $img_metadata
+  config_digest=$(docker manifest inspect -v $IMAGE | jq -r ".[] | select(.Descriptor.platform == {\"architecture\": \"$ARCH\", \"os\": \"$OS\"}) | .OCIManifest.config.digest + .SchemaV2Manifest.config.digest")
```

Comment:
Is `-v` needed here? The image digest is available without adding `-v` IIRC.

Reply:
`-v` is needed if you want the config digest, which is what we're extracting here. It's true that the image digest is available without `-v`.